docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -1,261 +1,462 @@
+to phase `init`. This means it has downloaded a new CA public key and is ready
+for state transitions.
 
-Verify that nodes have caught up:
+You can use the `tctl get` command with the following resources to determine
+the rotation state of the `host` CA on each agent kind:
 
-```code
-$ tctl get nodes --format=json | jq '.[] | {hostname: .spec.hostname, rotation: .spec.rotation.state, phase: .spec.rotation.phase}'
-{
-  "hostname": "terminal",
-  "rotation": "in_progress",
-  "phase": "update_servers"
-}
-```
+| Role                | `tctl get` value |
+|---------------------|------------------|
+| Application Service | `app_server`     |
+| Database Service    | `db_server`      |
+| Kubernetes Service  | `kube_server`    |
+| Proxy Service       | `proxies`        |
+| SSH Service         | `nodes`          |
 
-**Finish the rotation**
+During each phase of the `host` CA rotation, make sure all Agents and Proxy
+Service instances have completed the transition to target phase before
+proceeding to the next phase. We will explain the phases in [Step
+2](#step-2-4-start-a-manual-rotation).
 
-Before wrapping up, verify that you have not lost any nodes and can connect to them, for example:
+If you are joining Teleport processes to a cluster via the Teleport Auth Service
+using a [join token](../../agents/join-services-to-your-cluster/join-token.mdx),
+each Teleport process will need a CA pin to trust the Auth Service. The CA pin
+will change after each `host` CA rotation. Make sure you use the *new* CA pin
+when adding Teleport services after `host` CA rotation.
+
+### `user`
+
+The `user` CA issues a certificate when a user authenticates to Teleport. It
+also signs client certificates for the Windows Desktop Service and Teleport SSH
+Service. Teleport-protected servers and Windows desktops use these certificates
+to verify connecting clients.
+
+Once you have completed the rotation and reached the final `standby` phase,
+users who have signed into Teleport must reauthenticate to receive a user
+certificate from the new CA, otherwise Teleport client commands fail.
+
+If you have registered Windows desktops with Teleport, [follow the
+guide](../../desktop-access/active-directory-manual.mdx) to exporting the
+Teleport user CA so the Windows Desktop Service can authenticate to RDP hosts.
+Verify that you can connect to registered desktops throughout the rotation.
+
+### `db` and `db_client`
+
+The `db` and `db_client` CAs issue certificates that the Teleport Database
+Service uses to communicate with self-hosted databases. 
+
+The Teleport Database Service presents a certificate signed by the `db_client`
+CA when communicating with a self-hosted database, which an admin configures to
+trust certificates issued by the CA. Self-hosted databases present a certificate
+signed by the `db` CA, which the Database Service uses to verify that a database
+server is a genuine Teleport-protected resource.
```

Comment:
```suggestion
trust certificates issued by the CA. Self-hosted databases can be issued a certificate by the `db` CA, which the Database Service uses by default to verify that a database
server is a genuine Teleport-protected resource.
```

it's optional to use a `db` CA cert - they can their own custom CA to issue a cert for their databases, but they must configure the db service to trust their custom CA